### ISSUE TITLE
Maybe it's worth mentioning that US is equivalent to "every sequentially compact subset is sequentially closed"

### ISSUE THREAD

**Issue by Jianing-Song** [pi-base/data#594]
### Discussed in https://github.com/orgs/pi-base/discussions/591

<div type='discussions-op-text'>

<sup>Originally posted by **Jianing-Song** March 20, 2024</sup>
This is analogous to KC = "every compact subset is compact".

Proof: Suppose that $X$ is US. Let $C$ be a sequentially compact subset of $X$, then there is no sequence in $C$ converging to a point outside $C$, otherwise this sequence has a subsequence converging to a point in $C$ by sequential compactess, and we get a sequence with two different limits. 

Conversely, suppose that every sequentially compact subset is sequentially closed. Each singleton is sequentially compact, hence sequentially closed, which implies that $X$ is $T_1$. Suppose that there is a sequence $(x_n)$ converging to both $a\neq b$. By taking the open neighborhood $U=X\setminus\{b\}$ of $a$ we see that $x_n\neq b$ for all sufficiently large $n\ge N$, and $(x_n)_{n\ge N}\cup\{a\}$ that is not sequentially closed ($b$ is in its sequential closure).</div>

**Comment by prabau** [comment]
I assume you meant KC = "every compact subset is closed" ?

**Comment by StevenClontz** [comment]
I think this is a good perspective to add, but is this result in the literature or a forum like Math.StackExchange that we can cite? We generally want the results we include in the pi-Base to be aired out more widely than our GitHub.

**Comment by StevenClontz** [comment]
If it's not somewhere citable, I'm happy to ask the question on Math.StackExchange for @Jianing-Song to answer.

**Comment by Jianing-Song** [comment]
> I assume you meant KC = "every compact subset is closed" ?

Yes, sorry for the typo. Corrected.

**Comment by prabau** [comment]
@Jianing-Song This result is an interesting observation.  Whether it fits as a "pi-base theorem" is another question.  As you have seen, the information in pi-base consist of a list of properties, a list of theorems and a list of spaces.  The properties are unique up to equivalence.  The theorems are of the form [some set of properties or their negation ==> some property/its negation].  Now in topology there are many results that do not fit into this mold, and hence do not fit into pi-base.

At the most, since the new property "every sequentially compact subset is sequentially closed" is equivalent to US, it could possibly be mentioned as a sentence in [US](https://topology.pi-base.org/properties/P000099).  But even that could be debatable.  Some properties are described with several equivalent definitions.  For example, [Bair ](https://topology.pi-base.org/properties/P000064) or Alexandrov or ...  But these the equivalent defs are usually semantically close to one another.  So I am not sure in this case.

**Comment by Jianing-Song** [comment]
> @Jianing-Song This result is an interesting observation. Whether it fits as a "pi-base theorem" is another question. As you have seen, the information in pi-base consist of a list of properties, a list of theorems and a list of spaces. The properties are unique up to equivalence. The theorems are of the form [some set of properties or their negation ==> some property/its negation]. Now in topology there are many results that do not fit into this mold, and hence do not fit into pi-base.
> 
> At the most, since the new property "every sequentially compact subset is sequentially closed" is equivalent to US, it could possibly be mentioned as a sentence in [US](https://topology.pi-base.org/properties/P000099). But even that could be debatable. Some properties are described with several equivalent definitions. For example, [Baire](https://topology.pi-base.org/properties/P000064) or Alexandrov or ... But these the equivalent defs are usually semantically close to one another. So I am not sure in this case. We should discuss further.

Yes, I understand this situation. Thanks!

**Comment by StevenClontz** [comment]
@prabau yeah, I was viewing this as an edit to US's description, not a new property. But we need a citation for it outside our GitHub.
